Doc/library/sqlite3.rst

```diff
@@ -1431,13 +1431,26 @@ case-insensitively by name:
 .. literalinclude:: ../includes/sqlite3/rowclass.py
 
 
+.. _sqlite3-connection-context-manager:
+
 Using the connection as a context manager
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Connection objects can be used as context managers
-that automatically commit or rollback transactions.  In the event of an
-exception, the transaction is rolled back; otherwise, the transaction is
-committed:
+Connection objects can be used as context managers that automatically commit or
```

Comment:
This is just personal nitpickery, but I think this is better as a singular: "A Connection object....as a context manager".

Question: the docs say that you'll get a connection object *unless* you supply a factory (See `connect` method doc). My inclination would be to not elaborate on that later, and take as suggested (`:class:Connection`)

Reply:
Yes, both connections and cursors can be created using factories. It is required, of course, that the original classes are used as some kind of parent / base. I don't think we need to mention this here; it will not make the docs more clear.